samples/js-customer-registry/customer_view.proto

```diff
@@ -140,7 +140,7 @@ service CustomerByNameView {
 
   rpc IgnoreOtherEvents(google.protobuf.Any) returns (domain.CustomerState) {
```

Comment:
Might be better with the callout on the rpc line instead, like it is for <1> and <4>

```suggestion
  rpc IgnoreOtherEvents(google.protobuf.Any) returns (domain.CustomerState) { // <7>
```